fieldservice_vehicle_stock/models/stock_picking_type.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class StockPickingType(models.Model):
+    _inherit = "stock.picking.type"
+
+    require_vehicle_id = fields.Boolean(
+        string='Require Vehicle',
+        help="Stock Pickings with this enabled must have the vehicle "
+             "set before they can reserve quantities. ",
```

Comment:
This explanation is not totally clear for me.

Reply:
" must have *this* vehicle set" maybe?

Reply:
What about?
```
help="You must provide the vehicle to reserve those pickings."
```